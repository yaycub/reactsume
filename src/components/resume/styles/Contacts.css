.Contacts{
  display: flex;
  flex-direction: column;

  h3{
    font-size: 15px;
    font-weight: bold;
    margin-bottom: 10px;
  }

  li{
    font-size: 13px;
    margin-bottom: 5px;
  }

  span{
    font-weight: bold;
  }
}