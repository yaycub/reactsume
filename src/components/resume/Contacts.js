import React from 'react';
import PropTypes from 'prop-types';
import styles from './styles/Contacts.css';

const Contacts = ({ phoneNumber, email, github, linkedIn }) => {
  return (
    <ul className={styles.Contacts}>
      <h3>Contact Info</h3>
      <li><span>Phone Number:</span> {phoneNumber}</li>
      <li><span>Email:</span> {email}</li>
      <li><span>Github:</span> {github}</li>
      <li><span>LinkedIn:</span> {linkedIn}</li>
    </ul>
  );
};

Contacts.propTypes = {
  phoneNumber: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired,
  github: PropTypes.string.isRequired,
  linkedIn: PropTypes.string.isRequired
};

export default Contacts;
