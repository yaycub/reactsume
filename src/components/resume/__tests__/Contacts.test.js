import React from 'react';
import { shallow } from 'enzyme';
import Contacts from '../Contacts';
import data from '../../../resume.json';

describe('Contacts Component', () => {
  it('can render Contacts', () => {
    const wrapper = shallow(<Contacts {...data.contacts}/>);
    expect(wrapper).toMatchSnapshot();
  });
})
;
