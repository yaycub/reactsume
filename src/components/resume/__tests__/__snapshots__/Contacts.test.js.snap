// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts Component can render Contacts 1`] = `
<ul>
  <h3>
    Contact Info
  </h3>
  <li>
    <span>
      Phone Number:
    </span>
     
    (555)555-5555
  </li>
  <li>
    <span>
      Email:
    </span>
     
    hellothere@hi.com
  </li>
  <li>
    <span>
      Github:
    </span>
     
    /yaycub
  </li>
  <li>
    <span>
      LinkedIn:
    </span>
     
    /yaycub
  </li>
</ul>
`;
